fbgemm_gpu/include/fbgemm_gpu/fbgemm_cuda_utils.cuh

```diff
@@ -625,7 +625,11 @@ template <typename T, int ReduceWidth = kWarpSize>
 DEVICE_INLINE T warpReduceAllSum(T val) {
 #pragma unroll
   for (int mask = ReduceWidth / 2; mask > 0; mask >>= 1) {
+#ifdef __HIP_PLATFORM_HCC__
+    val += __shfl_xor(val, mask);
+#else
     val += shfl_xor(val, mask);
+#endif
```

Comment:
`shfl_xor()` is a wrapper to support both HIP and CUDA.
https://github.com/pytorch/FBGEMM/blob/main/fbgemm_gpu/include/fbgemm_gpu/fbgemm_cuda_utils.cuh#L614-L621

Do you still need this branch? If you need it, is it possible to update that wrapper?

Reply:
The branch is removed.